README.md

```diff
@@ -19,9 +19,12 @@ SLIC Watch provides a CloudWatch Dashboard and Alarms for:
  8. SNS
  9. EventBridge
 
-Currently, SLIC Watch is available as a Serverless Framework plugin. Serverless Framework v2 and v3 are supported.
+ Support:
```

Comment:
I would suggest removing the "Support" section and replace with a sentence.

"SLIC Watch is available for *Serverless Framework*, *AWS SAM* and *CloudFormation*.
- Serverless Framework v2 and v3 are supported in the SLIC Watch Serverless Plugin.
- SLIC Watch is available as a CloudFormation Macro published in the Serverless Application Repository (SAR). This allows you to add SLIC Watch to SAM or CloudFormation templates by simply adding a `Transform` to your template.

Reply:
Did we say that SAR makes it also available for CDK?

I have seen online that you can do something like this:

```typescript
new sam.CdnApplication(this, 'slicWatch', {
  location: {
    applicationId: 'arn:...',
    semanticVersion: 'x.y.z'
  },
  parameters: {}
})
```

Have we tried this? If yes and it works we should definitely state (in big capital letters) that we support CDK too 😄

Reply:
Yes, CDK supports macros, but the format to supply configuration metadata differs, so requires testing. Eoin suggested we release the support as a separate PR.

Reply:
Yes, @ssamueluk had an example CDK project before. I'm suggesting that we focus testing efforts on SAM and add CDK tests and documentation afterwards.